lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -138,6 +140,38 @@ window.runLighthouseInExtension = function(options, categoryIDs) {
     });
 };
 
+/**
+ * Run lighthouse for connection and provide similar results as in CLI.
+ * @param {!Connection} connection
+ * @param {string} url
+ * @param {!Object} options Lighthouse options.
+          Specify lightriderFormat to change the output format.
+ * @param {!Array<string>} categoryIDs Name values of categories to include.
+ * @return {!Promise}
+ */
+window.runLighthouseAsInCLI = function(connection, url, options, categoryIDs) {
+  // Default to 'info' logging level.
+  log.setLevel('info');
+  const startTime = Date.now();
+  return window.runLighthouseForConnection(connection, url, options, categoryIDs)
+    .then(results => {
+      const endTime = Date.now();
+      results.timing = {total: endTime - startTime};
+      const artifacts = results.artifacts;
```

Comment:
seems like you don't really need to make this var, assuming you change L164

Reply:
Done